operator/config/watcher/operator_v1alpha1_watcher.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     "operator.kyma-project.io/watched-by": "lifecycle-manager"
   field: "spec"
   serviceInfo:
-    name: klm-event-service
+    name: event-service
```

Comment:
`klm-` is the namePrefix for KCP deployment, if you replace it with `lifecycle-manager-event-service`, the name not fit.

Reply:
But in a non-KCP deployment scenario it doesn't work, since the prefix is `lifecycle-manager`. Whatever prefix it is should not be a part of this file.